apps/core/src/pages/404.tsx

```diff
@@ -26,22 +25,14 @@ export const StyledContainer = styled('div')(() => {
 
 export const NotFoundPage = () => {
   const t = useAFFiNEI18N();
-  const { jumpToSubPath, jumpToIndex } = useNavigateHelper();
+  const { jumpToIndex } = useNavigateHelper();
+
   return (
     <StyledContainer data-testid="notFound">
       <img alt="404" src="/imgs/invite-error.svg" width={360} height={270} />
 
       <p>{t['com.affine.notFoundPage.title']()}</p>
-      <Button
-        onClick={() => {
-          const id = localStorage.getItem('last_workspace_id');
-          if (id) {
-            jumpToSubPath(id, WorkspaceSubPath.ALL);
-          } else {
-            jumpToIndex();
-          }
-        }}
-      >
+      <Button onClick={() => jumpToIndex()}>
```

Comment:
useCallback

Reply:
I have seen lint error in CI, but why my local lint is successful.

Reply:
oh, u commit a patch to improve the code